### PR TITLE
Quick updates to cross-mio files

### PR DESCRIPTION
**Pull request by cdobs**
Updated:
-Cruiseinformation.csv to add AR-40 which was an existing cruise that cp_340 was recovered on
-sensor_bulk_load-AssetRecord.csv - during the cspp calibration review it was noted that one FLORTJ (SN 1205) needed to be updated in bulkload to have the BBFL2W appended to the name.

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1358,7 +1358,7 @@ CGINS-FLORTD-01487,,Sensor,0,3-Wavelength Fluorometer: FLORT Series D,FLORT SERI
...
 CGINS-FLORTJ-01084,N00593,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-001,WET Labs,ECO Triplet-w,BBFL2W-1084,,20130630,13013.52,,Oregon State University
 CGINS-FLORTJ-01156,OL000612,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-006,WET Labs,ECO Triplet-w,BBFL2W-1156,,20140929,13013.52,also N00601,Oregon State University
-CGINS-FLORTJ-01205,N00561,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-005,WET Labs,ECO Triplet-w,1205,,20140801,13013.52,,
+CGINS-FLORTJ-01205,N00561,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-005,WET Labs,ECO Triplet-w,BBFL2W-1205,,20140801,13013.52,,
 CGINS-FLORTJ-01206,N00585,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-002,WET Labs,ECO Triplet-w,BBFL2W-1206,,20140630,13013.52,,Oregon State University
 CGINS-FLORTJ-01207,N00577,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-003,WET Labs,ECO Triplet-w,BBFL2W-1207,,20140616,13013.52,,Oregon State University
 CGINS-FLORTJ-01208,N00569,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-004,WET Labs,ECO Triplet-w,BBFL2W-1208,,20140801,13013.52,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv]: Looks good.  Side note - at some point we need to bring the 15 or so FLORDMs and probably these FLORTJs in line w/ the majority of the other FLORD/Ts and remove the model designation from the S/Ns (exa BBFL2W).

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -178,4 +178,5 @@ RC20190827,R/V Rachel Carson,2019-08-27T12:00:00,2019-08-27T19:00:00,ce06issp_00
 SKQ201920S,R/V Sikuliaq,2019-09-19T16:10:00,2019-10-05T00:28:00,Papa 7
 AR-38,R/V Neil Armstrong,2019-09-20T14:00:00,2019-09-22T15:15:00,Pioneer glider recovery
 AR-39,R/V Neil Armstrong,2019-09-25T16:55:00,2019-10-15T16:26:00,Pioneer 13
-SKQ201921S,R/V Sikuliaq,2019-10-10T05:00:00,2019-10-24T00:00:00,Endurance 12
\ No newline at end of file
+SKQ201921S,R/V Sikuliaq,2019-10-10T05:00:00,2019-10-24T00:00:00,Endurance 12
+AR-40,R/V Neil Armstrong,2019-10-20T12:00:00,2019-11-06T16:00:00,Pioneer glider cp_340 recovery
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv]: Looks good.

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv]: Looks good.

**Pull request merged**